commit d28e9da5e097aa8e0b8b9d54030cc990e8801eb6
Author: bertin <macportsuser@macports.org>
Date:   Fri Oct 10 19:03:36 2025 +0200

    /opt/local/linux-ports/audio/audacity/files/patch-fix_bundlepy-@prefix@.diff

diff --git a/scripts/build/macOS/fix_bundle.py b/scripts/build/macOS/fix_bundle.py
index 18401bb14..182623435 100644
--- a/scripts/build/macOS/fix_bundle.py
+++ b/scripts/build/macOS/fix_bundle.py
@@ -4,7 +4,8 @@ import subprocess
 import re
 
 def is_system_lib(path):
-    return path.startswith('/System/Library/') or path.startswith('/usr/lib/')
+    ## RJVB/MacPorts: we're installing into @PREFIX@
+    return path.startswith('/System/Library/') or path.startswith('/usr/lib/') or path.startswith('@PREFIX@/lib/')
 
 
 def macos_collect_rpaths(file):
